.travis.yml

```diff
@@ -27,4 +27,4 @@ install:
   - python setup.py install
 
 script:
-  - python -m unittest discover --start-directory magpie/tests
+  - python -m unittest discover
```

Comment:
Agreed. So far though there's just one test, so I'm not sure if it's worth to introduce a new framework for that :-)